js/plugins/Header.jsx

```diff
@@ -31,6 +31,13 @@ export const Header = ({url = "/header/", page = "mapstore", height = 90, ignore
                 header.setAttribute("logo-url", logoUrl);
                 header.setAttribute("stylesheet", stylesheet);
                 headerScript.src = script;
+                if (stylesheet) {
```

Comment:
Maybe it could be interesting to use / loop over an array of stylesheet (to separate header stylesheet and mapstore stylesheet) ?